From eb5b1a444ffc1beb5fb580f6295a53680cdedc1f Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Tue, 20 Nov 2018 14:55:40 +0800
Subject: [PATCH 380/706] arm64: dts: ls1028a: Update audio nodes for
 LS1028AQDS board

This patch updates audio nodes for LS1028AQDS board.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 6b4ce31821d554cbb5d245829ea61fb3ad0a0c8f)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-qds.dts     | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
index f7571e1c6cb5..c5b1ec72a7a5 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
@@ -67,14 +67,14 @@
 	sys_mclk: clock-mclk {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-		clock-frequency = <24576000>;
+		clock-frequency = <25000000>;
 	};
 
-	reg_3p3v: regulator-3p3v {
+	reg_1p8v: regulator-1p8v {
 		compatible = "regulator-fixed";
-		regulator-name = "3P3V";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
+		regulator-name = "1P8V";
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
 		regulator-always-on;
 	};
 
@@ -94,7 +94,7 @@
 			"Speaker Ext", "LINE_OUT";
 
 		simple-audio-card,cpu {
-			sound-dai = <&sai2>;
+			sound-dai = <&sai1>;
 			frame-master;
 			bitclock-master;
 		};
@@ -103,7 +103,7 @@
 			sound-dai = <&codec>;
 			frame-master;
 			bitclock-master;
-			system-clock-frequency = <24576000>;
+			system-clock-frequency = <25000000>;
 		};
 	};
 };
@@ -204,8 +204,8 @@
 				#sound-dai-cells = <0>;
 				compatible = "fsl,sgtl5000";
 				reg = <0xa>;
-				VDDA-supply = <&reg_3p3v>;
-				VDDIO-supply = <&reg_3p3v>;
+				VDDA-supply = <&reg_1p8v>;
+				VDDIO-supply = <&reg_1p8v>;
 				clocks = <&sys_mclk>;
 			};
 		};
-- 
2.17.1

